shoes-core/spec/shoes/builtin_methods_spec.rb

```diff
@@ -106,4 +78,49 @@
       end
     end
   end
+
+  %w(alert confirm).each do |type|
+    describe '##{alert}' do
+     let(:dialog) { double('dialog') }
+     let(:message) { double('message') }
```

Comment:
Do we actually need these to be doubles? Seems like a simple string would be sufficient since that's what the actual arguments are passed as.